Gordon360/Services/RecIM/TeamService.cs

```diff
@@ -40,7 +40,6 @@ public TeamViewModel GetTeamByID(int teamID)
                                 Status = _context.TeamStatus
                                             .FirstOrDefault(ts => ts.ID == t.StatusID)
```

Comment:
```suggestion
                                            .FirstOrDefault(ts => ts.ID == t.StatusID)?
```

Reply:
Same below on line 50 pls.

Reply:
Can't actually do this due to a syntax error: an expression lambda tree can't contain a null propagating operator

Reply:
Ah, yes. I remember that one. @EjPlatzer do you have any ideas as to how we should mitigate the possible null return value warning if at all?

Reply:
> Ah, yes. I remember that one. @EjPlatzer do you have any ideas as to how we should mitigate the possible null return value warning if at all?

To be clear, the warning is not that the value of the expression might be null, it's that you might be dereferencing a null value - that is, `FirstOrDefault` might return `null`, but you are accessing the `Description` property of the value of `FirstOrDefault`. If you try to access a property of a null value, you get a runtime error.

The options I see are to extract this LINQ expression to a separate statement and handle the null cases more explicitly, or to tell the compiler that you're certain that `FirstOrDefault` will never evaluate to `null` here, via the `!` operator.